tmpl/partials/footer.html

```diff
@@ -52,7 +52,7 @@
       <hr class="ipw-sectioner" style="margin-bottom: 15px;" />
       <div class="col-xs-12">
         <div class="ipw-copyright">
-          &copy; Interplanetary Networks, Inc. |
+          &copy; IPFS Community |
```

Comment:
they do. we need to create an official org for IPFS, "IPFS Community" is not a legal entity yet, not sure that we can assign copyright otherwise -- (i'm looking into it later on -- creating a foundation), 

for now either make it "Protocol Labs" -- or remove the copyright notice entirely.